tldr/change_impact.py

```
@@ -192,19 +193,36 @@ def find_affected_tests(
             except ValueError:
                 affected_tests.add(str(abs_path))
 
+    # Build call graph once for all functions
+    call_graph = None
+    reverse_graph = None
+    if all_changed_functions:
+        try:
+            call_graph = build_project_call_graph(str(project), language=language)
+            reverse_graph = build_reverse_graph(call_graph.edges)
+        except Exception:
+            pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)